lisa/features/gpu.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+from enum import Enum
+from typing import Any
+
+from lisa.base_tools.wget import Wget
+from lisa.feature import Feature
+from lisa.operating_system import Redhat, Ubuntu
+from lisa.tools import Uname
+from lisa.util import LisaException, SkippedException
+from lisa.util.logger import get_logger
+
+FEATURE_NAME_GPU = "Gpu"
+
+ComputeSDK = Enum(
+    "ComputeSDK",
+    [
+        # GRID Driver
+        "GRID",
+        # CUDA Driver
+        "CUDA",
+    ],
+)
+
+# Link to the latest GRID driver
+# The DIR link is
+# https://download.microsoft.com/download/9/5/c/95c667ff-ab95-4c56-89e0-e13e9a76782d/NVIDIA-Linux-x86_64-460.32.03-grid-azure.run
+DEFAULT_GRID_DRIVER_URL = "https://go.microsoft.com/fwlink/?linkid=874272"
+
+
+class Gpu(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
+        super().__init__(node, platform)
+        self._log = get_logger("gpu", self.name(), self._node.log)
```

Comment:
s/"gpu"/FEATURE_NAME_GPU?

Reply:
Let me fix this.

Reply:
This is enough,
`self._log = get_logger("feature", self.name(), self._node.log)`

Reply:
For operating_systems, I see it is like this  - get_logger(name="os", parent=self._node.log)

so, what is the first param intended to be?

Reply:
first is the type, second is id (optional). For example,
`get_logger(name="os", parent=self._node.log)` -> os 
`get_logger(name="os", "1", parent=self._node.log)` -> os[1]

Reply:
Ok Done